### PR TITLE
Fix a couple type mismatches

### PR DESCRIPTION
**Pull request by jdinan**
*Issue #, if available:*

None

*Description of changes:*

Fix mismatched types.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ryanamazon — DISMISSED** [review]

**Comment by ryanamazon** [comment]
Thank you!

File: src/nccl_ofi_net.c [hunk 1]
```diff
@@ -119,7 +119,7 @@ static const char *nccl_ofi_req_state_str(nccl_ofi_req_state_t state)
 	}
 }
 
-static const char *nccl_ofi_req_direction_str(nccl_ofi_req_state_t state)
```
Comment (rashikakheria) [inline src/nccl_ofi_net.c:122]: nit: Could you also change the name of the variable to direction?

**Review by rashikakheria — COMMENTED** [review]
Looks good other than a minor comment.

File: src/nccl_ofi_net.c [hunk 2]
```diff
@@ -119,7 +119,7 @@ static const char *nccl_ofi_req_state_str(nccl_ofi_req_state_t state)
...
+static const char *nccl_ofi_req_direction_str(nccl_ofi_req_direction_t state)
```
Comment (jdinan) [inline src/nccl_ofi_net.c:122]:
```suggestion
static const char *nccl_ofi_req_direction_str(nccl_ofi_req_direction_t direction)
```

File: src/nccl_ofi_net.c [hunk 3]
```diff
@@ -119,7 +119,7 @@ static const char *nccl_ofi_req_state_str(nccl_ofi_req_state_t state)
...
 {
 	switch(state) {
```
Comment (jdinan) [inline src/nccl_ofi_net.c:124]:
```suggestion
	switch(direction) {
```

File: src/nccl_ofi_net.c (hunk 1 above)
Comment (jdinan) [reply src/nccl_ofi_net.c:122]: Done

**Comment by jdinan** [comment]
@rashikakheria Feedback addressed. This should be ready to merge. Thanks!

**Comment by rashikakheria** [comment]
> @rashikakheria Feedback addressed. This should be ready to merge. Thanks!

Thanks a lot James! Could you also merge the commits into one before the merge?

**Comment by jdinan** [comment]
Done

**Review by rashikakheria — APPROVED** [review]

**Pull request merged**